### DISCUSSION TITLE
Status of Revolt Chat.

### DISCUSSION THREAD

**Discussion by seythegamer** [stoatchat/discussions#724] [Q&A] [answered]
Dear devs,

Whats the status of rewriting the desktop app and audio functionality from scratch?

Me and my group of friends (10+) people are looking for alternative to Discord, because it began to be over-comercialized lately.

Im really looking forward to use your application, but in its todays form it not usable at all.
You can't grant permissions to be able to use voice-chat, when this function is the most neccesary for such app.
There's also no screenshare option in the desktop app.

All of us really hope that Revolt will start being functional.
if possible please write me back, and tell me what is the plus minus date in which the refreshed version of the application will be released. we really put our hopes in you.

**Comment by Rexogamer** [comment, marked as answer]
We don't have any solid dates at the moment and don't provide ETAs. That said, we're making decent progress with the new web/desktop apps - once these are done, we intend on focusing on getting voice support up to scratch.
